tck/src/main/java/org/eclipse/microprofile/openapi/apps/petstore/resource/PetResource.java

```diff
@@ -170,7 +173,7 @@ public Response deletePet(
                                     schema = @Schema(implementation = ApiResponse.class)))
     @RequestBody(name = "pet",
                  content = @Content(mediaType = "application/json", schema = @Schema(implementation = Pet.class),
-                                    examples = @ExampleObject(ref = "http://example.org/petapi-examples/openapi.json#/components/examples/pet-example")),
+                                    examples = @ExampleObject(ref = "#/components/examples/pet-example")),
```

Comment:
Using an external ref like this should be valid, right?

The resulting document won't validate because it points to an invalid URL, but this looks like the only test we have that an external reference from an annotation is passed through to the output.

Reply:
Restored back to the original external ref.